members/federal_us_veterans_sec_robert_wilkie.yaml

```diff
@@ -159,7 +159,7 @@ contact_form:
         required: false
       - name: message
         selector: textarea[id*='Threads']
-        value: $MESSAGE
+        value: "have a good day"
```

Comment:
We might want to use something that would catch their attention should it accidentally get through.
```
Attempting to workaround issues for contact form. If you see this message, please contact <our support email>
```

Reply:
Added